bitaxe_hashrate_benchmark.py

```
@@ -222,88 +335,135 @@ def restart_system():
+                            return False
+
+                        if vr_temp and vr_temp >= max_vr_temp:
+                            print(RED + f"CRITICAL: VR temperature {vr_temp}°C exceeded limit {max_vr_temp}°C during stabilization!" + RESET)
+                            return False
+                            
+                        if power and power > max_power:
+                            print(RED + f"CRITICAL: Power {power}W exceeded limit {max_power}W during stabilization!" + RESET)
+                            return False
+                            
+                except Exception:
+                    pass # Ignore transient errors during restart
```

Ruff (0.14.14)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)